README.md

```diff
@@ -1,11 +1,34 @@
 # react-scroll-sync
-Synced scroll position across multiple scrollable elements
 
-## Demo
+Synced scroll position across multiple scrollable elements.
 
-http://react-sync-scroll.netlify.com/
+## Installation
+
+```shell
+$ npm install react-scroll-sync
+```
+
+## Usage
+
+```html
+<ScrollSync>
+  <div style={{ display: 'flex', position: 'relative', height: 300 }}>
+    <ScrollSyncPane>
+      <div />
+    </ScrollSyncPane>
+
+    <ScrollSyncPane>
+      <div />
+    </ScrollSyncPane>
+      
+    <ScrollSyncPane>
+      <div />
+    </ScrollSyncPane>
+  </div>
+</ScrollSync>
+```
 
-## Documentation & Example
+## Documentation
```

Comment:
Why removing `Example`?

Reply:
We can let it but an example is a part of a documentation. Seems useless to me.

Reply:
Okay fair enough.